File: java/spring-arch-unit/spring-arch-unit-mvc-example/src/main/java/com/bmarques/springarchunitmvcexample/domain/invoice/Invoice.java
package com.bmarques.springarchunitmvcexample.domain.invoice;

import com.bmarques.springarchunitmvcexample.domain.customer.Customer;
import lombok.Builder;

import java.time.LocalDate;

@Builder
public class Invoice {

    protected int id;
    protected LocalDate installment;

    protected Double value;

    protected Customer customer;
}

File: kotlin/spring-arch-unit/spring-arch-unit-mvc-example/src/main/kotlin/com/bmarques/springarchunitmvcexample/domain/invoice/Invoice.kt
package com.bmarques.springarchunitmvcexample.domain.invoice

import com.bmarques.springarchunitmvcexample.domain.customer.Customer
import java.time.LocalDate

abstract class Invoice {
    abstract val id: Int
    abstract val installment: LocalDate
    abstract val value: Double
    abstract val customer: Customer
}
